pkg/providers/baremetal/mnnvl.go

```diff
@@ -95,67 +93,96 @@ func getIbTree(ctx context.Context, _ []string) (*topology.Vertex, error) {
 						ibKey := ibPrefix + strconv.Itoa(ibCount)
 						treeRoot.Vertices[ibKey] = ibRoot
 						break
+					} else {
+						fmt.Printf("Missing ibnetdiscover output\n")
```

Comment:
Should we use log instead?
Do we need to return error?
Do we need to have a metric for this error?

Reply:
I am not sure about this. It is not really an error scenario if IB is not present. This text is supposed to be informational. Should we remove it instead?